core/drivers/gic.c

```diff
@@ -124,6 +125,13 @@
 #define GICD_SGIR_NSATT_SHIFT			15
 #define GICD_SGIR_CPU_TARGET_LIST_SHIFT		16
 
+/* GICD ICFGR bit fields */
+#define ICFGR_TYPE_EDGE		2
```

Comment:
Please add a  `GICD_` prefix since the name of the register(s) starts with `GICD_ICFGR`